brave/brave-core
# Use Rust BLS implementation

Resolves https://github.com/brave/brave-browser/issues/20024

- Use Rust implementation for BLS public key calculation instead of typescript library

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Enable Filecoin feature in chrome://flags
- export bls account key from filecoin ganache
- import it to brave wallet
- compare addresses in wallet and ganache, they should be same
